### PR TITLE
Redesign bottom layout of DesignActivity

### PR DESCRIPTION
**Pull request by elfilibusterismo**
Redesign project file selector and replace BuildingDialog
https://github.com/user-attachments/assets/2cb4c69a-931b-471e-9c2b-720254426b30

### CODE REVIEW

File: app/src/main/java/com/besome/sketch/design/DesignActivity.java [hunk 1]
```diff
@@ -1284,10 +1281,8 @@ private void doInBackground() {
                     dialog.show();
                 });
             } catch (zy zy) {
-                isBuildFinished = true;
```
Comment (elfilibusterismo) [inline app/src/main/java/com/besome/sketch/design/DesignActivity.java:1184]: This should be included I think

File: app/src/main/java/com/besome/sketch/design/DesignActivity.java [hunk 2]
```diff
@@ -1284,10 +1281,8 @@ private void doInBackground() {
...
                 activity.indicateCompileErrorOccurred(zy.getMessage());
             } catch (Throwable tr) {
-                isBuildFinished = true;
```
Comment (elfilibusterismo) [inline app/src/main/java/com/besome/sketch/design/DesignActivity.java:1187]: Same here

File: app/src/main/java/com/besome/sketch/design/DesignActivity.java (hunk 1 above)
Comment (elfilibusterismo) [reply app/src/main/java/com/besome/sketch/design/DesignActivity.java:1184]: @ilyassesalama

Comment (ilyassesalama) [reply app/src/main/java/com/besome/sketch/design/DesignActivity.java:1184]: I believe those 2 lines weren't removed with the commit I just committed recently (?)

**Pull request merged**